test/efficiency-tests/activations.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+import time
+
+import syft as sy
+import torch as th
+
+
+@pytest.mark.parametrize("activation", ["tanh", "sigmoid"])
+def test_activation(activation, hook, workers):
+    bob = workers["bob"]
+    alice = workers["alice"]
+    crypto_prov = sy.VirtualWorker(hook, id="crypto_prov")
+
+    if activation == "tanh":
+        func = th.tanh
+    else:
+        func = th.sigmoid
+
+    x = th.randn([10, 10]).fix_precision().share(bob, alice, crypto_provider=crypto_prov)
+    t0 = time.time()
```

Comment:
Would be nice to modularize time monitoring. I suggest to create a function `assert_time_is_less_than(func, time=1)`, that implements this logic. This should be stored in another file at the efficiency-tests folders named `efficiency_assertions.py` (or something similar).

Reply:
Love this idea !

Reply:
Great idea!

Reply:
It would help benchmark different parts of our codebase and see how various optimization would help.

Reply:
Yeah, good idea! What do you think about a decorator for that?

Reply:
Yes!! Decorators are great for this.